pkg/cmd/taskrun/list.go

```diff
@@ -32,14 +34,23 @@ import (
 	cliopts "k8s.io/cli-runtime/pkg/genericclioptions"
 )
 
-const (
-	emptyMsg = "No TaskRuns found"
-)
+const listTemplate = `{{- $trl := len .TaskRuns.Items }}{{ if eq $trl 0 -}}
+No TaskRuns found
+{{- else -}}
+NAME	STARTED	DURATION	STATUS{{- if $.AllNamespaces }}	NAMESPACE{{- end }}
```

Comment:
I'm assuming that `{{- if $.AllNamespaces }}` is throwing off the spacing for the `NAMESPACE` header here. Maybe the header could be a variable and checking could be done in advance for `AllNamespaces`? When I run `tkn tr ls` locally, this is what I get:

```
NAME                STARTED        DURATION     STATUS               NAMESPACE
sleep1s-run-k6kxd   21 hours ago   17 seconds   Succeeded   default
sleep1s-run-4fd8q   21 hours ago   20 seconds   Succeeded   default
sleep1s-run-j2vwf   21 hours ago   16 seconds   Succeeded   default
```

I would always try to give this a run locally before pushing this up to be sure.

Reply:
I do run this locally before pushing :angel: 
Don't understand why there are inconsistencies in the output. I have used a tab instead of 6 spaces, I was unaware of that. WIll try it with that.